### DISCUSSION TITLE
how do we know if an events not related to my game

### DISCUSSION THREAD

**Discussion by essamce** [SFML/imgui-sfml#262] [Q&A] [answered]
hi, how do i know if ImGui has handled user input ?
code below shows how i imagined it will be looks like/

```c++
while (window.pollEvent(event)) 
{
            bool guiEvent = ImGui::SFML::ProcessEvent(window, event);  
            if (guiEvent ) continue; // that wa a gui event

           //  some other handlers

 }
```

**Comment by oprypin** [comment, marked as answer]
* I believe this is solved here https://github.com/SFML/imgui-sfml/issues/24#issuecomment-427003384

**Reply by essamce** [reply]
@oprypin
yes, thank you, that's exactly what i'm looking for,
